src/lib/core/components/filter/HistogramFilter.tsx

```diff
@@ -548,15 +548,15 @@ function HistogramPlotWithControls({
       truncationConfigIndependentAxisMax
     ) {
       setTruncatedIndependentAxisWarning(
-        'Data has been truncated (light gray area) by range selection'
+        ' Data has been truncated (light gray area) by range selection'
```

Comment:
I don't really like how these warnings read.  I'd prefer something like

```
Data has been truncated by range selection, as indicated in the plot by the light gray shading.
```
What do @danicahelb and @SheenaTomko think?

I know that's very long, but there is space for two lines of text in the blue notification area.  I tried a longer message and it will need some CSS changes so that it wraps and doesn't make the blue box much wider.

Reply:
@bobular will try. It would be a bit tricky to change CSS though

Reply:
@bobular As suggested I have made single space at Notification component so the single space at histogram filter is removed

Reply:
@bobular also adjusted notification size with the new message by setting style from histogram filter component. Like,

![histogramfilter-update1](https://user-images.githubusercontent.com/12802305/132563163-8db04208-b82d-4e19-ae7b-87361d1ca859.png)